modular_zubbers/code/modules/vending/multisec.dm

```diff
@@ -35,12 +38,21 @@
 					/obj/item/clothing/head/security_cap = 10,
 					/obj/item/clothing/head/beret/sec/peacekeeper = 6,
 					/obj/item/clothing/head/beret/sec/peacekeeper/white = 6,
+					/obj/item/clothing/head/cowboy/skyrat/cattleman/sec = 6,
+					/obj/item/clothing/head/cowboy/skyrat/cattleman/wide/sec = 6,
 					/obj/item/clothing/head/helmet/sec/terra = 6,
 					/obj/item/clothing/head/hats/warden/police/patrol = 6,
 					/obj/item/clothing/head/costume/ushanka/sec = 10,
 					/obj/item/clothing/gloves/color/black/security = 10,
 					/obj/item/clothing/head/helmet/sec/futuristic = 6,
 					/obj/item/clothing/suit/armor/vest/collared_vest = 6, //added by Bangle
+					/obj/item/clothing/accessory/armband/deputy/lopland = 6,
+					/obj/item/clothing/under/colonial/nri_police = 6,
+					/obj/item/clothing/suit/armor/vest/nri_police = 6,
+					/obj/item/clothing/neck/cloak/colonial/nri_police = 6,
+					/obj/item/clothing/head/hats/colonial/nri_police = 6,
```

Comment:
Rest of the PR looks good but it does feel a little weird to have all the NRI gear in the regular non-contraband section, could you move the NRI outfit to contraband?